components/launcher/src/server/mod.rs

```diff
@@ -16,7 +16,7 @@ mod handlers;
 
 use std::collections::HashMap;
 use std::fs;
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
```

Comment:
This got fixed as part of https://github.com/habitat-sh/habitat/pull/5689.  It might be worth rebasing this PR.

Reply:
Ah, cool; will do.